css/styles.css

```diff
@@ -1112,4 +1107,4 @@ input {
 	.journal-filter-time {
 		visibility: visible;
 	}
-}
+}
```

Comment:
It is good practice to have a final blank newline at the end of every text line to prevent concatenated output merging the last and first lines of files.